sparkmagic/proposal.md

```diff
@@ -0,0 +1,62 @@
+# Proposal for Incubation
+
+## Subproject name
+
+sparkmagic
+
+## Development team and Advocate
+
+Subproject development team:
+
+* Alejandro Guerrero Gonzalez, Microsoft (`@aggFTW`)
+* Sangeetha Shekar, Microsoft (`@sangeethashekar`)
+* Auberon Lopez, Cal Poly (`@alope107`)
+
+Steering Council Advocate
+
+* Brian Granger (`@ellisonbg`)
+
+## Subproject goals, scope and functionality
+
+Goals:
+* Provide the ability to connect from any IPython notebook to different remote Spark clusters.
+* Frictionless Spark usage from IPython notebook without a local installation of Spark.
+* The functionality delivered will be pip-installable or conda-installable.
+
+Scope:
+* IPython magics to enable remote Spark code execution through [Livy](https://github.com/cloudera/hue/tree/master/apps/spark/java), a Spark REST endpoint, which allows for Scala, Python, and R support as of September 2015.
+* The project will create a Python Livy client that will be used by the magics.
+* The project will integrate the output of Livy client with the rich visualization framework that is being proposed [LINK].
```

Comment:
I would clarify that pandas data frames will be used as an intermediate in this chain